packages/transaction_signing_gateway/lib/model/wallet_public_info.dart

```diff
@@ -1,9 +1,20 @@
 import 'package:equatable/equatable.dart';
 
 class WalletPublicInfo extends Equatable {
+  /// a user-specified name to distinguish different wallets
   final String name;
+
+  /// public hex address, for example "cosmos1ec4v57s7weuwatd36dgpjh8hj4gnj2cuut9sav" for cosmos-based chain
+  /// or "0x9BCf3021B356F9352D3811fFCd4F778D63FdF208" for ethereum-based chain
   final String publicAddress;
+
+  /// a uuid of the wallet credentials, should be generated by client apps when importing wallet info,
```

Comment:
```suggestion
  /// a uuid of the wallet credentials, must be generated by client apps when importing wallet info,
```

Reply:
oooh "should" is all sorts of fun! If a uuid is required, then we can use "must"
see https://developers.google.com/style/word-list#letter-s 
(the entry for should is pasted below)
should, should be
Generally avoid.
Because should is ambiguous by definition, it can be problematic. For example, if you're telling the reader what to do, should implies that the action is recommended but optional, leaving the reader unsure about what to do.
Clarify what you mean. Determine if an action is required versus optional, an outcome is expected versus possible, or a state is actual versus recommended.
An action is required. Use must, or rephrase the sentence so that it is a clear imperative instruction such as Do the following before you continue.
An action is recommended. Use We recommend... or Google recommends…. You can use should if a recommended action is generally recognized—for example, You should use a strong password…, or, You should follow the principle of least privilege....
An action is optional. Use can. For example, You can also use approach B to solve the same problem.
An outcome is expected. Describe the outcome in terms of what is expected, for example, The process returns 10 items.
An outcome is possible. Use might or can. For example, The process can take about 30 minutes.
A state is actual. When you're describing the state of something, such as the value of a variable, avoid writing The value should be true. Instead, clarify which of the following you mean:
You must set the value to true.
The server sets the value to true.
If the value is false, follow these steps to change it to true.